src/Knp/Rad/ResourceResolver/EventListener/ResourcesListener.php

```diff
@@ -38,6 +39,7 @@ public function resolveResources(FilterControllerEvent $event)
         foreach ($resources as $resourceKey => $resourceDetails) {
             $parameters = [];
 
+            $resourceDetails = array_merge(['required' => true], $resourceDetails);
```

Comment:
I think you can also set `arguments` to `[]` by default, and so remove the other `isset()`.